tg_bot/modules/cust_filters.py

```diff
@@ -145,14 +145,14 @@ def stop_filter(bot: Bot, update: Update):
 def reply_filter(bot: Bot, update: Update):
     chat = update.effective_chat  # type: Optional[Chat]
     message = update.effective_message  # type: Optional[Message]
-    to_match = extract_text(message)
-    if not to_match:
+    match_string = extract_text(message)
+    if not match_string:
         return
-
     chat_filters = sql.get_chat_triggers(chat.id)
     for keyword in chat_filters:
-        pattern = r"( |^|[^\w])" + re.escape(keyword) + r"( |$|[^\w])"
-        if re.search(pattern, to_match, flags=re.IGNORECASE):
+       try:
+         pattern = r"( |^|[^\w])" + keyword + r"( |$|[^\w])"
+         if re.search(pattern, match_string, flags=re.IGNORECASE).group():
```

Comment:
I think `if re.search(...):` would be enough because it will be None anyway if there's no match. This way you can get rid of that ugly try